source/documentation/deploying_services/redis.md

```diff
@@ -193,34 +195,34 @@ Type `yes` when asked for confirmation.
 
 You can store data classified up to ‘official’ on the GOV.UK PaaS. Refer to the [data security classification documentation](/deploying_services/#data-security-classification) for more information.
 
-### Redis plans
+### Redis service plans
+
+There are two service plans currently available for the Redis service:
 
-There are two plans available for the Redis service:
+- `tiny-3.2`
+- `medium-ha-3.2`
 
-- `tiny-clustered-3.2`
-- `tiny-unclustered-3.2`
+Do not use the `tiny-clustered-3.2` service plan for new service instances as we have deprecated this plan.
 
-You cannot migrate a service from an unclustered to a clustered plan or vice versa.
+Both service plans include encryption at rest of the database storage. This means that both the data on the disk and in snapshots is encrypted.
 
-Both plans include encryption at rest of the database storage. This means that both the data on the disk and in snapshots is encrypted.
+Amazon ElastiCache Redis backs up both service plans every day.
 
-#### tiny-clustered-3.2
+You can [vertically scale](/managing_apps.html#scaling) or upgrade your service plan.
 
-We recommend that you use this plan as it is backed up every day. Note that:
+#### Billing
 
-- [Sidekiq](https://sidekiq.org) [external link] does not work with clustered Redis
-- this plan cannot be [vertically scaled](/managing_apps.html#scaling) or upgraded to a bigger plan
+If your org is in its trial period, you can use the `tiny-3.2` service plan for free.
 
-#### tiny-unclustered-3.2
+Trial orgs cannot access paid service plans by default. If paid service plans are not enabled, when you try to use a paid service plan, you will receive an error stating “service instance cannot be created because paid service plans are not allowed”.
 
-Use this plan if your client library cannot connect to clustered plans. Note that:
+One of your Org Managers must contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) to request that we enable paid service plans.
```

Comment:
Does this apply to trial orgs or are there non-trial orgs that don't have paid service plans enabled?
If the latter applies, I think this information should be at the top of the section.